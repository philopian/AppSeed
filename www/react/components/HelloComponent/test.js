import React from "react";
import { configure, shallow, mount, render } from "enzyme";
import renderer from "react-test-renderer";
import "raf/polyfill";
import Adapter from "enzyme-adapter-react-16";
import Component from "./index.js";

describe("HelloComponent (Snapshot)", () => {
  it("HelloComponent renders without crashing", () => {
    const component = renderer.create(<Component />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});

describe("Addition", () => {
  it("knows that 2 and 2 make 4", () => {
    const actual = 2 + 2;
    const expected = 4;
    expect(actual).toEqual(expected);
  });
});
