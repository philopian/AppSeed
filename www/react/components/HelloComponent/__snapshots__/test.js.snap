// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloComponent (Snapshot) HelloComponent renders without crashing 1`] = `
<div
  className="my-component"
>
  <div
    className="sc-bdVaJa kSUpXQ"
  >
    <h1>
      HelloComponent
    </h1>
    <p>
      it 
      <span>
        World
      </span>
    </p>
  </div>
</div>
`;
